math/number_of_positive_divisors.cpp

```diff
@@ -25,46 +25,52 @@ respectively.
  *
 **/
 
+#include <cassert>
 #include <iostream>
 #include <vector>
 
 /**
  * Algorithm
  */
 int number_of_positive_divisors(int n) {
-    std::vector<int> prime_exponent_count;
+    if (n < 0) {
+        n = -n;
+    }
+
+    int divisors_count = 1;
     for (int i = 2; i * i <= n; i++) {
```

Comment:
```suggestion
    for (int i = 2; i * i <= n; i++) {
        // check for all positive divisors until sqrt(n)
```

Reply:
This is not completely true since the variable `n` is being modified. I'll think of a way how to document this functionality.